b2/console_tool.py

```diff
@@ -260,38 +207,45 @@ def __str__(self):
 
 
 class AuthorizeAccount(Command):
-    """
-    b2 authorize-account [<applicationKeyId>] [<applicationKey>]
-
-        Prompts for Backblaze applicationKeyId and applicationKey (unless they are given
-        on the command line).
-
-        You can authorize with either the master application key or
-        a normal application key.
-
-        To use the master application key, provide the application key ID and
-        application key from the "B2 Cloud Storage Buckets" page on
-        the web site: https://secure.backblaze.com/b2_buckets.htm
-
-        To use a normal application key, created with the create-key
-        command or on the web site, provide the application key ID
-        and the application key itself.
 
-        You can also optionally provide application key ID and application key
-        using environment variables {B2_APPLICATION_KEY_ID_ENV_VAR} and
-        {B2_APPLICATION_KEY_ENV_VAR} respectively.
-
-        Stores an account auth token in {B2_ACCOUNT_INFO_DEFAULT_FILE} by default,
-        or the file specified by the {B2_ACCOUNT_INFO_ENV_VAR} environment variable.
-
-        Requires capability: listBuckets
-    """
+    FORBID_LOGGING_ARGUMENTS = True
 
-    OPTION_FLAGS = ['dev', 'staging']  # undocumented
+    @classmethod
+    def add_subparser(cls, subparsers, parent):
+        parser = cls._add_parser(
+            subparsers,
+            """
```

Comment:
I think this should stay in `cls.__doc__` and it should just be referenced here. Same goes for every other Command. Is there any reason we need to remove the docstring from the class and keep it here? Also, in my design the code of `add_subparser` would be exactly the same for all classes, so it can use `super()` to run that line and then only add arguments